daml-script/test/src/test-utils/com/daml/lf/engine/script/test/AbstractFuncIT.scala

```diff
@@ -474,8 +474,10 @@ abstract class AbstractFuncIT
         }
       }
     }
+    // TODO https://github.com/digital-asset/daml/issues/15882
+    //  reactive when canton supports consortium party
     "ChoiceAuthority:test" should {
-      "succeed" in {
+      "succeed" ignore {
```

Comment:
@nickchapman-da  The test fail with an up-to-date canton.

Reply:
Awesome. I guess the default response to requesting authority, is _No!_

Reply:
```
Caused by: scala.NotImplementedError: an implementation is missing
	at scala.Predef$.$qmark$qmark$qmark(Predef.scala:344)
	at com.digitalasset.canton.platform.apiserver.services.RejectionGenerators$.processDamlException$1(RejectionGenerators.scala:61)
	at com.digitalasset.canton.platform.apiserver.services.RejectionGenerators$.processInterpretationError$1(RejectionGenerators.scala:157)
	at com.digitalasset.canton.platform.apiserver.services.RejectionGenerators$.processLfError$1(RejectionGenerators.scala:165)
	at com.digitalasset.canton.platform.apiserver.services.RejectionGenerators$.commandExecutorError(RejectionGenerators.scala:177)
	at com.digitalasset.canton.platform.apiserver.services.ApiSubmissionService.failedOnCommandExecution(ApiSubmissionService.scala:229)
	at com.digitalasset.canton.platform.apiserver.services.ApiSubmissionService.$anonfun$handleCommandExecutionResult$1(ApiSubmissionService.scala:149)
	at scala.util.Either.fold(Either.scala:190)
	at com.digitalasset.canton.platform.apiserver.services.ApiSubmissionService.handleCommandExecutionResult(ApiSubmissionService.scala:151)
	at com.digitalasset.canton.platform.apiserver.services.ApiSubmissionService.$anonfun$evaluateAndSubmit$1(ApiSubmissionService.scala:168)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:470)
	... 8 common frames omitted
```